nflow-engine/src/main/java/io/nflow/engine/workflow/instance/WorkflowInstance.java

```diff
@@ -75,6 +75,11 @@
    */
   public final String type;
 
+  /**
+   * The priority of the workflow instance. Defaults to 0 and can also be negative.
```

Comment:
Could you document which direction, higher or lower priority numbers gets more attention and processing power from the nflow?